ethereum/spec/src/main/java/tech/pegasys/teku/spec/config/builder/SpecConfigBuilder.java

```diff
@@ -102,21 +104,21 @@ public class SpecConfigBuilder {
   private Eth1Address depositContractAddress;
 
   // Networking
-  private Integer gossipMaxSize;
-  private Integer maxChunkSize;
-  private Integer maxRequestBlocks;
-  private Integer epochsPerSubnetSubscription;
-  private Integer ttfbTimeout;
-  private Integer respTimeout;
-  private Integer attestationPropagationSlotRange;
-  private Integer maximumGossipClockDisparity;
-  private Bytes4 messageDomainInvalidSnappy;
-  private Bytes4 messageDomainValidSnappy;
+  private Integer gossipMaxSize = 10485760;
+  private Integer maxChunkSize = 10485760;
+  private Integer maxRequestBlocks = 1024;
+  private Integer epochsPerSubnetSubscription = 256;
+  private Integer ttfbTimeout = 5;
+  private Integer respTimeout = 10;
+  private Integer attestationPropagationSlotRange = 32;
+  private Integer maximumGossipClockDisparity = 500;
+  private Bytes4 messageDomainInvalidSnappy = Bytes4.fromHexString("0x00000000");
+  private Bytes4 messageDomainValidSnappy = Bytes4.fromHexString("0x01000000");
   private Integer subnetsPerNode;
```

Comment:
Should `subnetsPerNode` have a default?

Reply:
this should probably follow the same pattern (whatever is decided below) i was mapping against another PR and missed this one...